project_administrator_restricted_visibility/tests/test_project_administrator_restricted_visibility.py

```diff
@@ -32,13 +31,25 @@ def setUpClass(cls):
                 "message_partner_ids": [(6, 0, cls.user_admin.ids)],
             }
         )
+        # Define some default field values
+        cls.env["ir.default"].create(
+            {
+                "field_id": cls.env.ref(
+                    "project.field_project_project__privacy_visibility"
+                ).id,
+                "json_value": '"followers"',
+            }
+        )
```

Comment:
suggestion: these defaults would affect all tests, but I think that they are only relevant for the use case you're covering. IMHO it would be better to include it inside the test itself, and not inside the `setUpClass`.